packages/core/admin/admin/src/content-manager/routes.tsx

```diff
@@ -1,5 +1,18 @@
 /* eslint-disable check-file/filename-naming-convention */
-import { Navigate, RouteObject } from 'react-router-dom';
+import { UID } from '@strapi/types';
+import { Navigate as ReactRouterNavigate, RouteObject, useLoaderData } from 'react-router-dom';
+
+const Navigate = () => {
+  const pathname = useLoaderData() as string;
+
+  return (
+    <ReactRouterNavigate
+      to={{
+        pathname,
+      }}
+    />
+  );
+};
```

Comment:
Why not just call it Redirect? isn't `useLoaderData` a generic? so you don't need to cast 🤔